pkgs/development/python-modules/httpie-ntlm/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, httpie
+, requests_ntlm
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "httpie-ntlm";
+  version = "1.0.2";
+  format = "setuptools";
+  disabled = pythonAtLeast "3.10";
```

Comment:
```suggestion
```
Why shouldn't it work with python 3.11? if we use httpie from the same pythonPackages it will always have the same python version.

Reply:
We don't want this. If we bump python3Packages to 3.10 it will silently break for no good reason.

Reply:
@SuperSandro2000 thanks for follow up :-) Maybe I have misunderstood, correct me if wrong. I have now removed the disable line. So now the package can be built as long `httpie` exist for given python version. I can see that all PR checks have passed. But `nixpkgs-review` fail for python 3.10 as expected.